benchmark/src/xts_benchmark_template.h

```diff
@@ -8,12 +8,12 @@
 {
 	char xts_algname[strlen(ALGNAME) + 5];
 	const size_t bufsize = g_params.bufsize;
-	u8 *orig = malloc(bufsize);
-	u8 *ctext = malloc(bufsize);
+	u8 *orig = calloc(1, bufsize);
```

Comment:
Why calloc not malloc?

Reply:
This is to avoid a compiler warning.  I've expanded the explanation in the commit message for this change.